collect_app/src/main/java/org/odk/collect/android/activities/CollectAbstractActivity.java

```diff
@@ -50,6 +54,7 @@
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         themeUtils = new ThemeUtils(this);
+        AppCompatDelegate.setDefaultNightMode(themeUtils.isDarkTheme() ? MODE_NIGHT_YES : MODE_NIGHT_NO);
```

Comment:
This sets the theme to light or dark mode based on the setting. We could also have this use the automatic mode, but that should probably be another issue/feature.